code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -429,8 +429,8 @@
 //	bitesize = 3													//This is the amount each bite consumes.
 
 /obj/item/weapon/reagent_containers/food/snacks/koisbar_clean
-	name = "k'ois bar"
-	desc = "Bland NanoTrasen produced K'ois bars, rich in syrup and injected with extra phoron."
+	name = "k'ois bar (unsafe for human consumption)"

```

Comment:
Remove the "(unsafe for human consumption)"
We do not have that anywhere else and something like that should go into the description

Reply:
Understandable. I guess we'll have to hope they read the description then.